wic/iot2050-swu.wks.in

```diff
@@ -17,4 +17,4 @@ part --source empty --align 1024 --fixed-size 4G --uuid "${ABROOTFS_PART_UUID_B}
 part /var --source rootfs --rootfs-dir=${IMAGE_ROOTFS}/var --label var --fstype=ext4 --mkfs-extraopts "-T default" --ondisk sda --align 1024  --size 512M
 part /home --source rootfs --rootfs-dir=${IMAGE_ROOTFS}/home --label home --fstype=ext4 --mkfs-extraopts "-T default" --ondisk sda --align 1024  --size 512M
 
-bootloader --ptable gpt --append "console=ttyS3,115200n8 earlycon=ns16550a,mmio32,0x02810000 rootwait watchdog.handle_boot_enabled=0 ${EXTRA_KERNEL_PARAMS}"
+bootloader --ptable gpt --append "${CONSOLE_KERNEL_PARAMS} rootwait watchdog.handle_boot_enabled=0 ${EXTRA_KERNEL_PARAMS}"
```

Comment:
Is that a "no new line at end of file"?

Reply:
@colpane , If this is fixed then we can merge it :)

Reply:
I can fix it, however all the wks.in files do not have a new line at the end of file and this wks.in file also did not have a new line at the end of file before I changed it :) Do I still need to add a new line?

Reply:
The affected file currently does have a newline at its end.

Reply:
Correct, it does not have, but other wks.in files (https://github.com/siemens/meta-iot2050/blob/master/wic/iot2050-swu-secure.wks.in) do not have it too. Why only this file should have the new line?

Reply:
Just don't remove the newline removal from your patch and resubmit. Avoid style regressions.

Reply:
Ah, sorry now I see the icon :) Thanks, fixed.